### ISSUE TITLE
yume does not work on clients

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#198]
**Reported by bli on 11 Jul 2006 06:33 UTC**
With trunk r5128 on Fedora Core 5, yume does not work on client nodes:

```
[~](root@oscarnode1)# yume install test
Unable to find node oscarnode1 in SIS database.
Usage: /usr/bin/yume OPTIONS [YUM_OPTIONS]

Call yum or repoquery with _only_ the repositories on the command line configured.
OPTIONS can be:
 --repo URL  : repository which should be included in yum actions
               (multiple --repo options are possible).
...
```

**Comment by dikim33** [comment]
**Comment by bli on 25 Jul 2006 03:11 UTC**
Culprit here seems to be ```scripts/distro-query```.  This patch (against r5196) seems to fix the problem.

To apply:

```
cd $OSCAR_HOME/scripts
patch -p0 < distro-query.patch
```

**Comment by dikim33** [comment]
**Comment by efocht on 25 Jul 2006 14:10 UTC**
While this patch seems to solve the issue with fc5, distro-query has two more problems:
 1. it wasn't fully adapted to the remote repository names
 2. it fails under SuSE because OSCAR_HOME won't be set in the ssh session invoked from the clients

Bernard, please check your patch ito trunk!

I have fixed issue 1. in trunk svn r5220.

Issue 2. needs a fix in the yume package.

All fixes need to be merged to branch-5-0!

**Comment by dikim33** [comment]
**Comment by efocht on 25 Jul 2006 14:13 UTC**
Oh, I already checked in fix for issue 2. into the yume source, but this didn't become an RPM yet. (svn r5219 pkgsrc/yume/trunk). Maybe yume gets some more updates before we make a new RPM...

**Comment by dikim33** [comment]
**Comment by bli on 27 Jul 2006 02:51 UTC**
Erich, I checked in my patch to trunk r5227 and branch-5-0 r5228.

**Comment by dikim33** [comment]
**Comment by efocht on 27 Jul 2006 14:55 UTC**
Merged fix from trunk r5220 to brach-5-0 r5230.

Fix for yume still needs to be done.

**Comment by dikim33** [comment]
**Comment by efocht on 19 Sep 2006 21:28 UTC**
(In [5356]) Merged commit r5352 by efocht from 2006-09-19 10:51:34 +0200 (Tue, 19 Sep 2006)
with svnmerge_1by1 version v1.1 31.7.2006
from repository https://svn.oscar.openclustergroup.org/svn/oscar/trunk
to repository https://svn.oscar.openclustergroup.org/svn/oscar/branches/branch-5-0
Original commit message was:

Attempt to fix bugs #199 and #258.

Closes #199
Closes #258
